test/pkg/environment/common/expectations.go

```diff
@@ -329,6 +329,17 @@ func (env *Environment) ConsistentlyExpectTerminatingPods(duration time.Duration
 	}, duration.String()).Should(Succeed())
 }
 
+func (env *Environment) ConsistentlyExpectLivePods(duration time.Duration, pods ...*corev1.Pod) {
```

Comment:
nit: do we want to rename this to `ConsistentlyExpectActivePods` to match the upstream nomenclature?

Reply:
Sure -- sounds fine to me.